client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,86 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <div>
+      <b-form-group
+        :label="$t('sanitizers.label')"
+        label-size="lg"
+      >
+        <template #label>
+          <div
+            class="d-flex"
+          >
+            Condition Fields
+
+            <b-button
+              variant="link"
+              class="p-0 ml-1 mr-auto"
+              @click="add"
```

Comment:
add what (better naming)

Reply:
![image](https://user-images.githubusercontent.com/31524469/213675066-7aa2ddcd-8847-4abf-8b0e-3e6ad60d01db.png)

This has kind of been the convention we have been using...

Reply:
Bad convention, it should be changed there aswell, since in the context of recordConfigurator add doesn't mean anything (are you adding a record?)